js/meetup.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './meetup/App';
+
+window.addEventListener('scroll', function() {
```

Comment:
I think this function should be in `componentDidMount` of  App component and `SECTION` (should be lowercase) should be used via use of refs.

Reply:
okay, @RohitRox  dai I will do that.